api/inventorypb/agents.proto

```diff
@@ -590,7 +590,7 @@ message ListAgentsResponse {
 
 message GetAgentRequest {
   // Unique randomly generated instance identifier.
-  string agent_id = 1 [(validator.field) = {string_not_empty: true}];
+  string agent_id = 1 [(validate.rules).string.min_len = 1];
```

Comment:
There is no support for `required` option for strings: It seems that required  is not supported for strings yet: https://github.com/bufbuild/protoc-gen-validate/issues/745